docs/FRSKY_R9.md

```diff
@@ -21,17 +21,17 @@ The Frsky R9M transmitter module and R9MM and R9MX receivers are commercially av
   </tbody>
 </table>
 
-Note that R9 hardware uses a slightly older LORA chip family, which doesn't support the 31 Hz mode. It also happens to be incompatible with the newer LORA chips when the specific spreading factor used by the mLRS 19 Hz mode is selected.  Unfortunately, this means that R9 hardware can't connect with the other mLRS boards which support 868/915 MHz.
+Note that R9 hardware can't connect with the other mLRS boards which support 868/915 MHz. This is because it uses the SX127x LoRa chipset, which happens to be incompatible with the newer SX126x LoRa chipset when the SF6 spreading factor used by the mLRS 19 Hz mode is selected.  The older chipset also doesn't support SF5 which is required for the mLRS 31 Hz mode.
 
 ## R9M Tx Module ##
 
 The R9M transmitter module is somewhat limited with respect to serial ports. It provides access to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
 
 1. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to re-power the module to make any change effective.
 
-2. One needs to use a "Frsky inverter" dongle to connect a standard serial adapter. You either can buy one or build it yourself.
+2. One needs to somehow (un)invert the serial signals for most uses. You can buy or build a "Frsky inverter" dongle to connect a standard serial adapter.
 
-3. If you use the mLRS Lua configuration script for configuration and thus don't need the CLI for configuration, you can avoid the inverter dongle and use the one of the supported ESP32 boards with the mlrs-wifi-bridge sketch to connect via MAVLink wirelessly to a Ground Control Station.  The ESP32 can directly use the inverted serial signals.
+3. If you use the mLRS Lua configuration script for configuration and thus don't need the CLI for configuration, you can avoid the inverter dongle and just use the one of the supported ESP32 boards with the mlrs-wifi-bridge sketch to connect via MAVLink wirelessly to a Ground Control Station.  The ESP32 can directly use the inverted serial signals.
```

Comment:
maybe "... you can avoid the inverter dongle when you use one of the supported ..."